src/domain/accounts/services/Cargo.toml

```diff
@@ -47,7 +48,9 @@ uuid = { version = "1", default-features = false }
 
 [dev-dependencies]
 kamu-accounts = { workspace = true, default-features = false, features = ["testing"] }
+kamu-auth-rebac-services = { workspace = true }
```

Comment:
Even though these are dev-dependencies, please double check we didn't create a loop here.